packages/surveys/src/components/general/Survey.tsx

```diff
@@ -37,6 +38,7 @@ export const Survey = ({
   startAtQuestionId,
   clickOutside,
   shouldResetQuestionId,
+  isMultiLanguageAllowed,
```

Comment:
I would like to keep the surveys package independent from our Web App limitations and would remove `isMultiLanguageAllowed` here.
In the Web App it's fine to limit the functionality with these flags but the external packages (js, js-core, surveys) should just render the survey given to them.
I think it should work just removing this and show the language switch when the survey has multiple languages set up.